packages/storage-driver-cloudinary/src/index.ts

```diff
@@ -119,10 +119,12 @@ export class DriverCloudinary implements Driver {
 		return `Basic ${base64}`;
 	}
 
-	async read(filepath: string, range?: Range) {
+	async read(filepath: string, options?: ReadOptions) {
+		const { range, version } = options ?? {};
+
 		const resourceType = this.getResourceType(filepath);
 		const fullPath = this.fullPath(filepath);
-		const signature = this.getParameterSignature(fullPath);
+		const signature = version !== undefined ? `v${version}` : this.getParameterSignature(fullPath);
```

Comment:
Should we add a comment here about the "version" being used to bust the cache? Potentially linking back to https://cloudinary.com/documentation/advanced_url_delivery_options#asset_versions